README.md

```diff
@@ -0,0 +1,101 @@
+Automation wrapper for iozone
+
+Description:
+  
+Location of underlying workload:
```

Comment:
Missing description of workload, and location of workload.   If there is no great "location" for the workload, just don't include the `Location of underlying workload` line